ember-electron/backend/controller.js

```diff
@@ -23,8 +23,14 @@ const fs = require('fs');
 class Controller {
     constructor(sysdigPath) {
         this.sysdigPath = sysdigPath || path.join(__dirname, '../resources/sysdig/');
-        this.sysdigExe = this.sysdigPath + 'sysdig';
-        this.csysdigExe = this.sysdigPath + 'csysdig';
+
+        if (process.platform === 'win32') {
```

Comment:
I'd probably make this check to define the "program name", and then unify the way to create the two "exe" properties.